<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPlans</class>
 <widget class="QDialog" name="AddPlans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>221</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>221</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>221</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TimeTask - Додати план</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/images/timetask-ico.ico</normaloff>:/images/timetask-ico.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 255, 188);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Початок плану:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Кінець плану:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Виберіть захід:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(215, 132, 0);
border-style: solid;
border-radius: 7px;
}
QPushButton:hover {
background-color: black;
color: rgb(215, 132, 0);
}
</string>
   </property>
   <property name="text">
    <string>Додати план в захід</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
background-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-color: rgb(215, 132, 0);
selection-color: rgb(0, 0, 0);
}
QComboBox::drop-down {
background-color: rgb(215, 132, 0);
}
QComboBox::down-arrow {
image: url(:/images/down.png);
top: 1px;
left: 1px;
}
QComboBox::drop-down:pressed {
background-color: rgb(154, 92, 0);
 }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>План:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(215, 132, 0);
border-style: solid;
border-radius: 7px;
}
QPushButton:hover {
background-color: black;
color: rgb(215, 132, 0);
}
</string>
   </property>
   <property name="text">
    <string>Закрити</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-color: rgb(215, 132, 0);</string>
   </property>
   <property name="lineWrapColumnOrWidth">
    <number>0</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateTimeEdit {
background-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-color: rgb(215, 132, 0);
}
QDateTimeEdit::up-button {
background-color: rgb(215, 132, 0);
}
QDateTimeEdit::up-arrow {
image: url(:/images/up.png);
}
QDateTimeEdit::down-button {
background-color: rgb(215, 132, 0);
}
QDateTimeEdit::down-arrow {
image: url(:/images/down.png);
}
QDateTimeEdit::up-button:pressed {
background-color: rgb(154, 92, 0);
 }
QDateTimeEdit::down-button:pressed {
background-color: rgb(154, 92, 0);
 }</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateTimeEdit {
background-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-color: rgb(215, 132, 0);
}
QDateTimeEdit::up-button {
background-color: rgb(215, 132, 0);
}
QDateTimeEdit::up-arrow {
image: url(:/images/up.png);
}
QDateTimeEdit::down-button {
background-color: rgb(215, 132, 0);
}
QDateTimeEdit::down-arrow {
image: url(:/images/down.png);
}
QDateTimeEdit::up-button:pressed {
background-color: rgb(154, 92, 0);
 }
QDateTimeEdit::down-button:pressed {
background-color: rgb(154, 92, 0);
 }</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
